source/MissionAction.cpp

```diff
@@ -240,9 +243,24 @@ bool MissionAction::CanBeDone(const PlayerInfo &player) const
 		if(flagship)
 			available += flagship->OutfitCount(it.first);
 		
-		// If the gift "count" is 0, that means to check that the player has at
-		// least one of these items.
-		if(available < -it.second + !it.second)
```

Comment:
Ah... yes, and you've also brought attention to something bigger I missed. I assumed that when a mission was saved out, it saved the `require` block... but it actually saves `require <outfit>` back to the file as `outfit <outfit> 0`. I'll need a separate save block. And yep, for backward compatibility with missions still in progress, I'll need your bit too.